src/sgame/sg_main.cpp

```diff
@@ -602,6 +603,12 @@ void G_InitGame( int levelTime, int randomSeed, bool inClient )
 
 	// Initialize build point counts for the intial layout.
 	G_UpdateBuildPointBudgets();
+
+	for ( int i = TEAM_NONE + 1; i < NUM_TEAMS; ++i )
```

Comment:
Remove this. The cvar callback should run when it is first initialized.

Reply:
Removing this actually breaks the PR

Reply:
The cvar callback does run for the default value. The problem is this happens before G_Init which memsets the global structs.